eessi_bot_job_manager.py

```diff
@@ -43,6 +43,27 @@
 
 from pyghee.utils import log, error
 
+AWAITS_LAUCH = "awaits_lauch"
+FAILURE = "failure"
+FINISHED_JOB_COMMENTS = "finished_job_comments"
+NEW_JOB_COMMENTS = "new_job_comments"
+MISSING_MODULES = "missing_modules"
+MULTIPLE_TARBALLS = "multiple_tarballs"
+NO_MATCHING_TARBALL = "no_matching_tarball"
+NO_SLURM_OUT = "no_slurm_out"
+NO_TARBALL_MESSAGE = "no_tarball_message"
+RUNNING_JOB = "running_job"
+RUNNING_JOB_COMMENTS = "running_job_comments"
+SLURM_OUT = "slurm_out"
+SUCCESS = "success"
+
+REQUIRED_CONFIG = {
+    NEW_JOB_COMMENTS: [AWAITS_LAUCH],
+    RUNNING_JOB_COMMENTS: [RUNNING_JOB],
+    FINISHED_JOB_COMMENTS: [SUCCESS, FAILURE, NO_SLURM_OUT, SLURM_OUT, MISSING_MODULES,
```

Comment:
Do we really need to make this required in the bot configuration file?

Why don't we just fall back to a default comment if the bot isn't configured with a custom comment?

Reply:
Created #176